repo/packages/J/jenkins/10/marathon.json.mustache

```diff
@@ -4,6 +4,8 @@
   "mem": {{service.mem}},
   "instances": 1,
   "env": {
+      "JENKINS_AGENT_ROLE": "{{#roles.jenkins-agent-role}}{{roles.jenkins-agent-role}}{{/roles.jenkins-agent-role}}{{^roles.jenkins-agent-role}}*{{/roles.jenkins-agent-role}}",
```

Comment:
Since a default value is set, can't this just be the value of `roles.jenkins-agent-role`? Similar to below and `acceptedResourceRoles`.